tracim/tracim/model/serializers.py

```diff
@@ -388,12 +388,14 @@ def serialize_node_for_page(content: Content, context: Context):
             is_new=content.has_new_information_for(context.get_user()),
             content=data_container.description,
             created=data_container.created,
+            updated=content.last_revision.updated,
             label=data_container.label,
             icon=ContentType.get_icon(content.type),
-            owner=context.toDict(data_container.owner),
+            owner=context.toDict(content.first_revision.owner),
+            last_author=context.toDict(content.last_revision.owner),
             status=context.toDict(data_container.get_status()),
             links=[],
-            revisions=context.toDict(sorted(content.revisions, key=lambda v: v.created, reverse=True)),
```

Comment:
I added it to count the number of revisions inside the getone.mak but since I serialize the number of revisions in the list, I no longer need the whole list of revisions.